ponder.schema.ts

```diff
@@ -13,6 +13,8 @@ export default createSchema((p) => ({
     expiry: p.bigint(),
     bid: p.string(),
     ask: p.string(),
+    recipient: p.bigint(),
+    value: p.bigint(),
```

Comment:
Could you move lines 16,17 with recipient and value to the line 14 and 15 between expiry and bid? If you check the documentation in the contract, this is how they are ordered